docs/guidesdraft/01-getting-started.md

```diff
@@ -0,0 +1,138 @@
+## Getting started
+
+### Introduction
+
+[CARTO VL](https://github.com/cartodb/carto-vl) is a JavaScript library to create custom location intelligence applications that leverage the power of [CARTO](https://carto.com/). It uses [WebGL](https://www.khronos.org/webgl/) to enable powerful vector maps.
```

Comment:
Why a link to khronos, is it a business? Maybe we could point to a more recognized / neutral source like https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API ?

Reply:
It is the official documentation afaik.